### ISSUE TITLE
Build failure with PG11: error: too few arguments to function 'BackgroundWorkerInitializeConnection'

### ISSUE THREAD

**Issue by ChristophBerg** [pgpartman/pg_partman#226]
Hi,
while trying to get the Debian packages ready for PG11, I noticed pg_partman doesn't support it yet:
```
13:19:05 gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -g -g -O2 -fdebug-prefix-map=/build/postgresql-11-bbCkV4/postgresql-11-11~beta2=. -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -fPIC -I. -I./ -I/usr/include/postgresql/11/server -I/usr/include/postgresql/internal  -Wdate-time -D_FORTIFY_SOURCE=2 -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include/mit-krb5  -c -o src/pg_partman_bgw.o src/pg_partman_bgw.c
13:19:05 src/pg_partman_bgw.c: In function 'pg_partman_bgw_run_maint':
13:19:05 src/pg_partman_bgw.c:398:5: error: too few arguments to function 'BackgroundWorkerInitializeConnection'
13:19:05      BackgroundWorkerInitializeConnection(dbname, pg_partman_bgw_role);
13:19:05      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
A new release with a fix would be welcome. Thanks!

**Comment by keithf4** [comment]
Yes, I'm aware that some work needs to be done for PG11. I usually wait until there's at least a release candidate to dive into the C code stuff because I've previously run into backend code changes during dev/beta forcing me to have to keep redoing fixes.

Appreciate that you're already working ahead to test things out!

**Comment by devrimgunduz** [comment]
Hi @keithf4 ,

When will you release a new package? Beta3 is already out and I started rolling the RPMs.

Thanks!

**Comment by keithf4** [comment]
I'm actually working on new features for 11, so this fix won't get rolled out until those are ready. Which will probably be a while.

**Comment by keithf4** [comment]
This has been fixed in version 4.0.0.
